### ISSUE TITLE
Provide explicit commands for increasing / decrease window zoom level

### ISSUE THREAD

**Issue by basarat** [alm-tools/alm#146]
Needed by people that want to embed this editor :rose:

**Comment by basarat** [comment]
Had a go at this. The following code gets a lot done: 

```ts
/**
 * Commands to manage the browser zoom level
 */
import * as commands from "../../commands/commands";
import * as ui from "../../ui";

/** TODO: move to redux */
let currentZoomLevel = 1;

// init
currentZoomLevel = parseInt(document.body.style.zoom || '1');

const changeFactor = 0.2;

commands.increaseZoomLevel.on(() => {
    // We don't check exact numbers due to floating point 🌹
    if ((currentZoomLevel + changeFactor) > 3) {
        ui.notifyInfoQuickDisappear('Already at maximum zoom allowed');
        return;
    }
    currentZoomLevel = currentZoomLevel + changeFactor;
    pushToDom();
});

commands.decreaseZoomLevel.on(() => {
    // We don't check exact numbers due to floating point 🌹
    if ((currentZoomLevel - changeFactor) < 0.6) {
        ui.notifyInfoQuickDisappear('Already at minimum zoom allowed');
        return;
    }
    currentZoomLevel = currentZoomLevel - changeFactor;
    pushToDom();
});


function pushToDom(){
    /**
     * Becareful of which method we use. e.g. `zoom: 1`
     *  worked beautifully but monaco mouse handling broke with that
     *  Hence using css transforms now
     */
    document.body.style.zoom = currentZoomLevel.toString();

    /** We need this to trigger the relayout handlers in our code base */
    window.dispatchEvent(new Event('resize'));
}
```

But change the `body.style.zoom` breaks monaco's handling of the `mouse click - > document location` handling. I tried CSS transform with scale as well but that results literal upscalling which looks aweful. So there is no nice way to do this with pure browser apis without having some undesireable sideeffects.

Looking at the source code for `vscode`(https://github.com/Microsoft/vscode/blob/ad5ff01ac46264bd13cf8b03cf43bd4af540c898/src/vs/workbench/electron-browser/actions.ts#L176) It essentially offloads it to electron's `webFrame.setZoomLevel` [docs](https://github.com/electron/electron/blob/master/docs/api/web-frame.md#webframesetzoomlevellevel) 

So in short, there isn't a nice browser level way, leaving it up to the embedder to check the docs of the embedding environment (e.g. as pointed out electron provides an API) :rose:
